frontend/pages/about/our-team.js

```diff
@@ -165,26 +165,32 @@ export default function OurTeam() {
                     return (
                       <div key={index} className={styles.execRow}>
                         {row.map((exec, index) => {
+                          const {
```

Comment:
Same as [this](https://github.com/unswwit/backend/pull/28#discussion_r962298921)